lib/fizzy/capi.cpp

```diff
@@ -360,28 +429,33 @@ inline FizzyExportDescription wrap(const fizzy::Export& exp) noexcept
 }  // namespace
 
 extern "C" {
-bool fizzy_validate(const uint8_t* wasm_binary, size_t wasm_binary_size)
+bool fizzy_validate(const uint8_t* wasm_binary, size_t wasm_binary_size, FizzyError* error)
```

Comment:
Hmm, why aren't these functions marked noexcept? It doesn't matter as it is `extern "C"`?

Reply:
They can't be, because they're not `noexcept` in `fizzy.h`

Reply:
The workaround we had in evmone/hera is by adding a NOEXCEPT macro in the C header: https://github.com/ewasm/hera/blob/master/include/hera/hera.h

Reply:
If you do decide to do this, can you do it in a new PR?

Reply:
Added in #775.